src/HotChocolate/AspNetCore/src/AspNetCore/Serialization/DefaultHttpResponseFormatter.cs

```diff
@@ -119,6 +122,18 @@ public class DefaultHttpResponseFormatter : IHttpResponseFormatter
             response.ContentType = format.ContentType;
             response.StatusCode = statusCode;
 
+            if (result.ContextData is not null &&
```

Comment:
@michaelstaib the HTTP tests are failing, since the ContextData on the Result is `null`:

![image](https://user-images.githubusercontent.com/45513122/208084307-5d1e3585-0a46-42df-a158-60ed22de5684.png)

Currently I'm adding the context value on the `IRequestContext.ContextData` in the request middleware, since I can't modify the `ContextData` on the `IRequestContext.Result`, because it's a `IReadOnlyDictionary`:

![image](https://user-images.githubusercontent.com/45513122/208084448-460cb95a-c71d-44a1-98f6-86fc69ffea9b.png)

Should the `ContextData` from the `IRequestContext` be copied to the `Result` and this is a bug, or how should I transport the header value from the Cache (a request middleware) to the transport layer (the DefaultHttpResponseFormatter)?

Besides this everything should compile now and be ready to go.